app/views/directives/add-mobile-client.html

```diff
@@ -0,0 +1,34 @@
+<div class="panel panel-default">
```

Comment:
The dialog should follow Patternfly markup and styles.

Here is an example from the console:

https://github.com/openshift/origin-web-console/blob/master/app/views/modals/confirm-save-log.html